cmd/v2upgrade.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/chanzuckerberg/fogg/config"
+	v1 "github.com/chanzuckerberg/fogg/config/v1"
+	"github.com/chanzuckerberg/fogg/errs"
+	"github.com/chanzuckerberg/fogg/plan"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	v2upgrade.Flags().StringP("config", "c", "fogg.json", "Use this to override the fogg config file.")
+	rootCmd.AddCommand(v2upgrade)
```

Comment:
I think we should just call this "upgrade" or something, so that it is more generic and we can add more stuff to it over time.